tests/infrared/16/enable-stf.yaml.template

```diff
@@ -20,9 +20,9 @@ custom_templates:
         ManagePolling: true
         ManagePipeline: true
 
-        # required to set valid parameter due to typo in ceilometer-write-qdr.yaml
-        # and will be resolved in a future release
+        # enable Ceilometer metrics and events
         CeilometerQdrPublishMetrics: true
+        CeilometerQdrPublishEvents: true
 
```

Comment:
Just an FYI that this should go away in the 16.2 timeframe since the changes upstream should be pulled in during full import. It's required for 16.1 because the ceilometer-write-qdr.yaml template is not in sync with upstream Train (missing these parameters).